client/components/map.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useMemo, useState } from 'react';
+import { GoogleMap, useLoadScript, Marker } from '@react-google-maps/api';
+
+export default function HomeMap() {
+  const { isLoaded } = useLoadScript({
+    googleMapsApiKey: 'AIzaSyC3q7FbfvJEgmnvm2Lq2dMKurjfBFqWQb0'
```

Comment:
You're going to want to move this into your `.env` file. Then update your webpack config.

```shell
GOOGLE_MAPS_API_KEY=asdfasdfasdfasdfasdfasdf
```

https://github.com/Learning-Fuze/final-project-instructions/blob/main/code-base.md#webpackconfigjs

Reply:
oops... added the rest of them